nixos/maintainers/scripts/ec2/amazon-image.nix

```diff
@@ -43,7 +43,7 @@ in {
 
     sizeMB = mkOption {
       type = with types; either (enum [ "auto" ]) int;
-      default = 2048;
+      default = 2252;
```

Comment:
need to say why this is needed but basically: our closure size increased and unsure this is working on master at all